crypto/bio/bss_dgram.c

```diff
@@ -50,6 +50,17 @@
 # define M_METHOD_RECVFROM   3
 # define M_METHOD_WSARECVMSG 4
 
+# if defined(__GLIBC__) && defined(__GLIBC_PREREQ)
+#  if !(__GLIBC_PREREQ(2, 14))
+#   undef NO_RECVMMSG
```

Comment:
why the undef here?  Its just going to get reset on line 61, no?

Reply:
To avoid the compiler complaining about it being redefined in the event it is already defined e.g. because OpenSSL was configured like this:

````
$ ./config -DNO_RECVMMSG
````